addons/account/models/account_move.py

```diff
@@ -407,6 +407,7 @@ def _onchange_invoice_date(self):
             if accounting_date != self.date:
                 self.date = accounting_date
                 self._onchange_currency()
+            self._onchange_recompute_dynamic_lines()
```

Comment:
@smetl does it look good to you?

Reply:
Could we move forward with this PR? We have a customer affected by this.